Update the whitelist and support request email address to cite Zendesk

issyl0 opened the pull request:
- The team behind the transition-dev@ mailing list was dissolved a few
  months ago in the GOV.UK restructure. The email address is still
  active and they still receive sporadic whitelist requests for
  domains. These requests are now fielded via second-line support, and
  so should go to Zendesk. Given that people using the Transition Tool
  are inside government departments or agencies, they should know what
  Zendesk is and how to send queries via it, without us having to link
  explicitly to it.

jamiecobbett commented:
I think if we can provide a link to Zendesk, we should provide a link.

rjc123 commented:
+1 to links instead of text.

adammaddison commented:
Just wanted to add a comment for old times' sake...

issyl0 commented:
@rjc123 Hello, stranger!

rgarner commented:
Build's failing. But +1 to links.

issyl0 commented:
OK. Following conversations, we should be linking to the Support app because that's where internals users submit requests that then get funnelled into our Zendesk. I've amended that commit to take account of this and add a link to the correct section within the support app - "general requests". Now I just need to work out how to make YAML locale files do HTML - http://guides.rubyonrails.org/i18n.html#using-safe-html-translations isn't working.

issyl0 commented:
@rgarner HELLO! :sunny:

jennyd commented:
:wave: to all you lovely people :rainbow: :link:

jennyd commented:
I wonder when @fofr, @davidmann and @fatbusinessman are going to join in...

fatbusinessman commented:
> I wonder when @fofr, @davidmann and @fatbusinessman are going to join in...

You called?

issyl0 commented:
The build on this is still failing. I'll fix it tomorrow morning once I've worked out what's going on with links in YAML files, as per an earlier comment... :confounded:

issyl0 commented:
The tests now pass - I found the right combination of `.html_safe` and added it to the right file. On Jenny's advice I also added a Cucumber feature to check that the flash message actually has a link in it...

![screen shot 2015-04-23 at 16 15 26](https://cloud.githubusercontent.com/assets/355033/7301304/d1aee43c-e9d9-11e4-9295-fdc10a20552d.png)

fofr commented:
Yay :fireworks:

adammaddison commented:
\o/

On 24 April 2015 at 10:34, Paul Hayes <notifications@github.com> wrote:

> Yay [image: :fireworks:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/transition/pull/488#issuecomment-95870934>.
>

issyl0 commented:
Goodbye, remnant of the old team... :cloud:

jamiecobbett commented:
![200_s](https://cloud.githubusercontent.com/assets/93739/7316556/cc756f68-ea6f-11e4-844b-c13307e4ae6f.gif)

rjc123 commented:
![images](https://cloud.githubusercontent.com/assets/2803445/7316649/9bdc1b44-ea70-11e4-93b1-797f4f87898b.jpg)

rgarner commented:
![](http://i.giphy.com/ADvd7XGR09IC4.gif)
